src/app/learn/index.html

```diff
@@ -1,2 +1,3 @@
 <app-nav></app-nav>
-<app-learn></app-learn>
\ No newline at end of file
+<app-learn></app-learn>
+<stache-back-to-top [offset]=200></stache-back-to-top>
```

Comment:
Implement the `@TypeConverter()` decorator so that consumers can type `offset="200"`.

(Also: put `200` in double quotes.)